packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -654,6 +650,8 @@ export class Grapher
     }
 
     @action.bound populateFromQueryParams(params: GrapherQueryParams): void {
+        this.externalQueryParams = omit(params, GRAPHER_QUERY_PARAM_KEYS)
```

Comment:
We want to update the external params every time we update normal grapher params. This is important for keeping multi-dim params up to date in the link to the grapher page (e.g. in the share menu).